packages/xrpl/src/models/transactions/common.ts

```diff
@@ -50,17 +57,37 @@ function isSigner(obj: unknown): boolean {
   )
 }
 
+const XRP_CURRENCY_SIZE = 1
+const ISSUE_SIZE = 2
 const ISSUED_CURRENCY_SIZE = 3
+const XCHAIN_BRIDGE_SIZE = 4
 
 function isRecord(value: unknown): value is Record<string, unknown> {
   return value !== null && typeof value === 'object'
 }
 
+/**
+ * Verify the form and type of an IssuedCurrency at runtime.
+ *
+ * @param input - The input to check the form and type of.
+ * @returns Whether the IssuedCurrency is properly formed.
+ */
+export function isIssue(input: unknown): input is IssuedCurrency {
```

Comment:
```suggestion
export function isIssue(input: unknown): input is Currency {
```

Input type should be `Currency` since an Issue can either be an `IssuedCurrency` or `XRP` type.